### DISCUSSION TITLE
Just Installed, and keeps telling me File format not supported

### DISCUSSION THREAD

**Discussion by darkjedeye** [openbroadcaster/observer#32] [Show and tell]
I have just installed the server on a vm for testing and familiarizing jan 22 08:28.

When i try and upload a media file, which is an MP# and has been set in admin as allowed. I get a message  saying "This file format is not supported"

![Audio admin](https://user-images.githubusercontent.com/29452733/72870784-cb129900-3cf1-11ea-83e3-15f8be481208.PNG)
![File not supported](https://user-images.githubusercontent.com/29452733/72870790-cd74f300-3cf1-11ea-92ad-3fe70f0ab08b.PNG)

hope these images help...

and hoping it's just a file permission error or something i may have skipped

**Comment by radiorob** [comment]
dependency issue fixed in Ver 5.1

**Comment by darkjedeye** [comment]
Thanks so much, will test out later today.

**Comment by PurpleNinja225** [comment]
What was the fix for the issue? I just installed Mar 13th and also have the same issue. trying to upload video files in the .avi format. using ubuntu server 18.04

**Comment by PurpleNinja225** [comment]
I am using ffmpeg instead of libav-tools. I fixed this issue by running the commands in the dependencies.txt

ln -s /usr/bin/ffmpeg /usr/local/bin/avconv
ln -s /usr/bin/ffprobe /usr/local/bin/avprobe

**Comment by Neepawa** [comment]
> What was the fix for the issue? I just installed Mar 13th and also have the same issue. trying to upload video files in the .avi format. using ubuntu server 18.04

To upload MP4 files, you need to go into **Admin > Media Settings** and select **MOV (Apple Quicktime).** The reason is a library OpenBroadcaster uses to identify files (`extras/getid3`) returns  *Quicktime/MOV* as the file type for MP4 files.
